tools/ci/docker-login.ps1

```diff
@@ -6,9 +6,19 @@ If ($lastExitCode -ne "0") {
     throw "Previous command returned $lastExitCode"
 }
 # DockerHub login
-$DOCKER_REGISTRY_LOGIN = $(& "C:\mnt\tools\ci\aws_ssm_get_wrapper.ps1" "$Env:DOCKER_REGISTRY_LOGIN_SSM_KEY")
-$DOCKER_REGISTRY_PWD = $(& "C:\mnt\tools\ci\aws_ssm_get_wrapper.ps1" "$Env:DOCKER_REGISTRY_PWD_SSM_KEY")
+$tmpfile = [System.IO.Path]::GetTempFileName()
+& "C:\mnt\tools\ci\aws_ssm_get_wrapper.ps1" "$Env:DOCKER_REGISTRY_LOGIN_SSM_KEY" "$tmpfile"
+If ($lastExitCode -ne "0") {
+    throw "Previous command returned $lastExitCode"
+}
+$DOCKER_REGISTRY_LOGIN = $(cat "$tmpfile")
+& "C:\mnt\tools\ci\aws_ssm_get_wrapper.ps1" "$Env:DOCKER_REGISTRY_PWD_SSM_KEY" "$tmpfile"
+If ($lastExitCode -ne "0") {
+    throw "Previous command returned $lastExitCode"
+}
+$DOCKER_REGISTRY_PWD = $(cat "$tmpfile")
 docker login --username "${DOCKER_REGISTRY_LOGIN}" --password "${DOCKER_REGISTRY_PWD}" "docker.io"
 If ($lastExitCode -ne "0") {
     throw "Previous command returned $lastExitCode"
 }
+Remove-Item "$tmpfile"
```

Comment:
```suggestion
$DOCKER_REGISTRY_PWD = $(cat "$tmpfile")
Remove-Item "$tmpfile"
docker login --username "${DOCKER_REGISTRY_LOGIN}" --password "${DOCKER_REGISTRY_PWD}" "docker.io"
If ($lastExitCode -ne "0") {
    throw "Previous command returned $lastExitCode"
}
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Feel free to ignore